app/views/shared/_back_to_login.html.erb

```diff
@@ -0,0 +1,3 @@
+<div data-go-back-link='true' style='display:none;line-height:3'>
```

Comment:
I would add a class (styling data attributes is probably a bad idea in general).

Here are the list of css modules we load. https://github.com/sul-dlss/sul-requests/tree/master/app/assets/stylesheets/modules

I would say that `forms.scss` or `buttons.scss` are logical places for this to live.